common/src/main/java/com/wynntils/models/worlds/BombModel.java

```diff
@@ -114,11 +128,17 @@ public BombInfo getLastBomb() {
     private static final class ActiveBombContainer {
         private final Map<BombKey, BombInfo> bombs = new ConcurrentHashMap<>();
 
-        public void add(BombInfo bombInfo) {
+        public void add(BombInfo bombInfo, boolean replaceIfExists) {
```

Comment:
Can you make this method private, and add a public `add(BombInfo)` and `forceAdd(BombInfo) methods? Reading code that has booleans is hard, because you don't know what the boolean does.